packages/nextcloud/lib/src/utils/date_time.dart

```diff
@@ -1,22 +1,25 @@
+import 'package:timezone/timezone.dart' as tz;
+
 /// Common methods to work with [DateTime] data.
 extension DateTimeUtils on DateTime {
-  /// Constructs a new [DateTime] instance
+  /// Constructs a new [tz.TZDateTime] instance
   /// with the given [secondsSinceEpoch].
   ///
-  /// If [isUtc] is false then the date is in the local time zone.
+  /// If [location] is not specified then the date is in the 'UTC' time zone.
   ///
-  /// The constructed [DateTime] represents
+  /// The constructed [tz.TZDateTime] represents
   /// 1970-01-01T00:00:[secondsSinceEpoch]Z in the given
-  /// time zone (local or UTC).
+  /// time zone (location or UTC).
   /// ```dart
   /// final newYearsDay =
-  ///     DateTimeUtils.fromSecondsSinceEpoch(1641031200, isUtc:true);
+  ///     DateTimeUtils.fromSecondsSinceEpoch(1641031200, location:tz.UTC);
   /// print(newYearsDay); // 2022-01-01 10:00:00.000Z
   /// ```
-  static DateTime fromSecondsSinceEpoch(int secondsSinceEpoch, {bool isUtc = false}) {
-    final millisecondsSinceEpoch = secondsSinceEpoch * Duration.millisecondsPerSecond;
+  static tz.TZDateTime fromSecondsSinceEpoch(int secondsSinceEpoch, {tz.Location? location}) {
+    location ??= tz.UTC;
```

Comment:
The package has the location as the first parameter and as required: https://github.com/srawlins/timezone/blob/dd7fef499aec4d3f851eb394fdefb9aa095ad911/lib/src/date_time.dart#L181
We should use the same API.